test/user_test.exs

```diff
@@ -41,7 +41,6 @@ defmodule ConfigCat.UserTest do
           email: "EMAIL",
           country: "COUNTRY",
           custom: %{
-            :atom_property => "ATOM_VALUE",
```

Comment:
I'm a bit worried about this change. This is going to force users to use `"key" => "value"` in their custom properties maps rather than the more idiomatic Elixir convention of `key: "value"`. Are we OK with that?

Reply:
Sure, I can revert it. 👍